### ISSUE TITLE
License free Account

### ISSUE THREAD

**Issue by JonasGreu** [FortAwesome/Font-Awesome#20488]
Hey Gittis, 
I´ve just created a free fontawesome account in order to implement some free icons on our website. Did any of you had some troubles with the usage regarding the license? Do I need a pro account in order to use the icons publicy?

Regards Jonas

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

I'll try to answer but please wait for a confirmation by @robmadole and maybe check the FAQs: https://fontawesome.com/plans

> Did any of you had some troubles with the usage regarding the license?

I guess there are no problems, except with if your application is using hosted kits and they exceed the monthly page view limit (which I don't actually remember, but it can be checked at https://fontawesome.com/kits).

There should be no problems at all if you self-host the assets

> Do I need a pro account in order to use the icons publicy?

No need at all

@robmadole

**Comment by JonasGreu** [comment]
Hey,

thank you for your quick answer!

Our website won´t surpass the 10.000 clicks per month, so that shouldn´t be an issue.

I also want to implement an icon in an email-template, I didn´t find anything regarding that problem. Do you have any experience or similar cases?

Regards Jonas

**Comment by tagliala** [comment]
> I didn´t find anything regarding that problem. Do you have any experience or similar cases?

I would avoid that, you can try to convert an icon to png and add it as attachment in the email in order to be used in the email itself

But this is not related to Font Awesome

Ref:
- https://css-tricks.com/a-guide-on-svg-support-in-email/
- https://mjml.io/

**Comment by JonasGreu** [comment]
So if I´m implementing the icon as png, I got no problem with the usage regarding the license?

**Comment by tagliala** [comment]
> So if I´m implementing the icon as png, I got no problem with the usage regarding the license?

Let's ask on #7976

**Comment by sensibleworld** [comment]
Yes, under the free license, you can convert the icons to PNG and use them. The free icons are under CC 4.0 https://creativecommons.org/licenses/by/4.0/
